src/background/telemetry.js

```diff
@@ -2,121 +2,189 @@
+      record_on_release: true,
+    },
+  };
   try {
-    browser.telemetry.registerEvents(TELEMETRY_CATEGORY, {
-      "startup": {
-        methods: ["startup"],
-        objects: ["addon", "webextension"],
-        extra_keys: [],
-      },
-      "iconClick": {
-        methods: ["iconClick"],
-        objects: ["toolbar"],
-        extra_keys: [],
-      },
-      "displayView": {
-        methods: ["render"],
-        objects: ["manage", "doorhanger"],
-        extra_keys: [],
-      },
-      "itemAdding": {
-        methods: ["itemAdding"],
-        objects: ["manage"],
-        extra_keys: [],
-      },
-      "itemUpdating": {
-        methods: ["itemUpdating"],
-        objects: ["manage"],
-        extra_keys: [],
-      },
-      "itemDeleting": {
-        methods: ["itemDeleting"],
-        objects: ["manage"],
-        extra_keys: [],
-      },
-      "itemAdded": {
-        methods: ["itemAdded"],
-        objects: ["manage"],
-        extra_keys: ["itemid"],
-      },
-      "itemUpdated": {
-        methods: ["itemUpdated"],
-        objects: ["manage"],
-        extra_keys: ["itemid"],
-      },
-      "itemDeleted": {
-        methods: ["itemDeleted"],
-        objects: ["manage"],
-        extra_keys: ["itemid"],
-      },
-      "itemSelected": {
-        methods: ["itemSelected"],
-        objects: ["manage", "doorhanger"],
-        extra_keys: [],
-      },
-      "addClick": {
-        methods: ["addClick"],
-        objects: ["manage"],
-        extra_keys: [],
-      },
-      "datastore": {
-        methods: ["added", "updated", "deleted"],
-        objects: ["datastore"],
-        extra_keys: ["itemid", "fields"],
-      },
-      "feedback": {
-        methods: ["feedbackClick"],
-        objects: ["manage"],
-        extra_keys: [],
-      },
-      "faq": {
-        methods: ["faqClick"],
-        objects: ["manage"],
-        extra_keys: [],
-      },
-      "itemCopied": {
-        methods: ["usernameCopied", "passwordCopied"],
-        objects: ["manage", "doorhanger"],
-        extra_keys: [],
-      },
-    });
+    browser.telemetry.registerEvents(TELEMETRY_CATEGORY, events);
   } catch (e) {
     if (e.message === "Attempt to register event that is already registered.") {
       // eslint-disable-next-line no-console
-      console.log("telemetry events already registered; skipping registration");
+      console.log("Telemetry events already registered; skipping registration.");
     } else {
       throw e;
     }
   }
+}
 
+function registerScalars() {
+  const scalars = {
+    // Current login count, set when the user opens the doorhanger or manage page.
+    "loginCount": {
+      kind: "count",
+      keyed: false,
+      record_on_release: true,
+    },
+
+    // not implemented yet
+
+    // Total number of logins copied or revealed during the current session
+    // TODO: increment a counter with each copy and each reveal
```

Comment:
thanks for planning to put this in, I know its extra work but its a valuable metric to have.